src/types/events-types.ts

```diff
@@ -23,10 +23,10 @@ export type EventsRecordsFilter = {
   dateCreated?: RangeCriterion;
 };
 
-export type EventsQueryFilter = EventsMessageFilter | EventsRecordsFilter | ProtocolsQueryFilter;
+export type EventsFilter = EventsMessageFilter | EventsRecordsFilter | ProtocolsQueryFilter;
```

Comment:
Maybe its a naming thing, but I got confused... like:
- What's the point of select few property for `EventsMessageFilter` if `EventFilter` accept `EventsRecordsFilter` and `ProtocolsQueryFilter`?

Can you add comments to both `EventsFilter` and `EventsMessageFilter` to show the differences?